Packages/Account/Sources/Account/AccountDetailView.swift

```diff
@@ -487,6 +483,15 @@ public struct AccountDetailView: View {
   }
 }
 
+private extension View {
+  func applyAccountDetailsRowStyle(theme: Theme) -> some View {
```

Comment:
I'm open to suggestions how to implement/name it differently.

Reply:
Name is fine!